src/reducers/Material.js

```diff
@@ -12,13 +12,15 @@ import {
     MATERIALS_GENERATE_SURFACE_FOR_ONE,
     MATERIALS_GENERATE_SUPERCELL_FOR_ONE,
     MATERIALS_SET_BOUNDARY_CONDITIONS_FOR_ONE,
+    MATERIALS_SET_IS_NON_PERIODIC_FOR_ONE,
 } from "../actions";
 
 function materialsUpdateOne(state, action) {
     const materials = state.materials.slice();  // get copy of array
     const index = action.index || state.index;  // not passing index when modifying currently displayed material
     const material = action.material.clone();   // clone material to assert props re-render
     material.isUpdated = true;                  // to be used inside components
+    material.isNonPeriodic = action.material.isNonPeriodic;
```

Comment:
This line is being addressed in SOF-5214, the `action.material.clone()` function does not currently clone the value of `isNonPeriodic`.

Reply:
Let's remove it here then, or put a link to how it is addressed in SOF-5214